### DISCUSSION TITLE
which xml file in dictionary folder corresponding to the multiple replace?

### DISCUSSION THREAD

**Discussion by faon-92** [SubtitleEdit/subtitleedit#7119] [Q&A] [answered]
Hi,

Does someone know, as I state in the title, which xml file in dictionary folder correspond to the multiplace replace in the program, when I unzip the portable version de SE?

thanks for your feedback,

**Comment by niksedk** [comment, marked as answer]
The "Multiple replace rules" are in the Settings.xml file (in the SE data folder - press ctrl+alt+shift+D in the SE main window to open this folder)

**Comment by faon-92** [comment]
great, thanks a lot

On Thu, Jul 20, 2023 at 10:21 AM Nikolaj Olsson ***@***.***>
wrote:

> The "Multiple replace rules" are in the Settings.xml file (in the SE data
> folder - press ctrl+alt+shift+D in the SE main window to open this folder)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/SubtitleEdit/subtitleedit/discussions/7119#discussioncomment-6496690>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A6MWTDHVCUTBVPXYONLUSK3XRDS7PANCNFSM6AAAAAA2OSS5AE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>


-- 
Feng Sridi
